jsdoc.json

```diff
@@ -7,7 +7,9 @@
         "includePattern": ".+\\.js$",
         "excludePattern": "(node_modules(\\\\|/))"
     },
-    "plugins": [],
+    "plugins": [
```

Comment:
mixed whitespaces and tabs

Reply:
I wonder why we have spaces here but our editorconfig enforces tabs. I'll convert to tabs.